medmodels/medrecord/types.py

```diff
@@ -87,6 +87,26 @@
 #: A type alias for input to a Pandas DataFrame for edges.
 PandasEdgeDataFrameInput: TypeAlias = Tuple[pd.DataFrame, str, str]
 
+#: A type alias for input to a node.
+NodeInput = Union[
+    NodeTuple,
+    Sequence[NodeTuple],
```

Comment:
```suggestion
    List[NodeTuple],
```
as `NodeTuple` is covariant